### PR TITLE
About

### LINKED ISSUE #376: Credit skycrypt and texture packs
As we pull our images through from SkyCrypt we should credit them and also the original creators of the textures.
Use something like:
```
The item textures are provided by <a href="https://sky.shiiyu.moe/">SkyCrypt</a>.
Internally they use the work of these amazing people.

<a href="https://minecraft.net/" target="_blank" rel="noreferrer">Vanilla </a><br>
<div>by <span>Mojang</span></div>

<a href="https://hypixel.net/threads/4101579" target="_blank" rel="noreferrer">FurfSky Reborn <small>v1.3</small></a><br>
<div>by <span>The Reborn Team</span></div>

<a href="https://hypixel.net/threads/3470904" target="_blank" rel="noreferrer">RNBW+ <small>v0.6</small></a><br>
<div>by <span>rainbowcraft2</span></div>

<a href="https://hypixel.net/threads/2103515" target="_blank" rel="noreferrer">Hypixel Skyblock Pack <small>v13</small></a><br>
<div>by <span>Packs HQ</span></div>

<a href="https://hypixel.net/threads/2138599" target="_blank" rel="noreferrer">FurfSky+ <small>v1.71</small></a><br>
<div>by <span>Furf__</span></div>

<a href="https://hypixel.net/threads/2147652" target="_blank" rel="noreferrer">Vanilla+ <small>v1.39</small></a><br>
<div>by <span>TBlazeWarriorT</span></div>
```

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #376

### CODE REVIEW

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

File: src/pages/About/About.tsx [hunk 1]
```diff
@@ -40,15 +40,40 @@ function ItemDetails() {
...
+
+                        <p><a href="https://hypixel.net/threads/2138599" target="_blank" rel="noreferrer">FurfSky+ <small>v1.71</small></a> by Furf__</p>
+
+                        <p><a href="https://hypixel.net/threads/2147652" target="_blank" rel="noreferrer">Vanilla+ <small>v1.39</small></a> by TBlazeWarriorT</p>
+                    </div>
+                    <hr />
+                    <h3>Data / API</h3>
+                    <p><a href="https://hypixel.net">Hypixel</a></p>
+                    <hr />
+                    <h3>Legal</h3>
```
Comment (Ekwav) [inline src/pages/About/About.tsx:71]:
```suggestion
                    <h2>Legal</h2>
<p>We are not affiliated with Mojang nor HyPixel</p>
```

File: src/pages/About/About.tsx [hunk 2]
```diff
@@ -40,15 +40,40 @@ function ItemDetails() {
...
                     <p>This is an OpenSource project developed on <a href="https://github.com/matthias-luger/hypixel-react">GitHub</a></p>
-                    <p>You can support this site by letting it opened and <b>disabling your adblocker</b>. Arc will use some of your bandwith and pay us for that. You can learn more by clicking on the blue icon in the lower left corner. (only there if you disabled your adblocker)</p>
-                    <p>Player Heads from <a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
-                    <p>Item icons from <a href="https://sky.lea.moe">sky.lea.moe</a></p>
-                    <p>Data from <a href="https://hypixel.net">Hypixel</a></p>
+                    <hr />
+                    <h3>Our API</h3>
+                    <p><a href="https://sky.coflnet.com/api">https://sky.coflnet.com/api</a></p>
+                    <hr />
+                    <h3>Player Heads</h3>
```
Comment (Ekwav) [inline src/pages/About/About.tsx:47]:
```suggestion
<h2>Credits</h2>
                    <h3>Player Heads</h3>
```

File: src/pages/About/About.tsx [hunk 3]
```diff
@@ -40,15 +40,40 @@ function ItemDetails() {
...
+                    <p><a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
```
Comment (Ekwav) [inline src/pages/About/About.tsx:48]:
```suggestion
                    <p>Are provided by<a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
```

File: src/pages/About/About.tsx [hunk 4]
```diff
@@ -40,15 +40,40 @@ function ItemDetails() {
                 <hr />
                 <Card className="about-card">
                     <p>This is an OpenSource project developed on <a href="https://github.com/matthias-luger/hypixel-react">GitHub</a></p>
-                    <p>You can support this site by letting it opened and <b>disabling your adblocker</b>. Arc will use some of your bandwith and pay us for that. You can learn more by clicking on the blue icon in the lower left corner. (only there if you disabled your adblocker)</p>
-                    <p>Player Heads from <a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
-                    <p>Item icons from <a href="https://sky.lea.moe">sky.lea.moe</a></p>
-                    <p>Data from <a href="https://hypixel.net">Hypixel</a></p>
+                    <hr />
+                    <h3>Our API</h3>
+                    <p><a href="https://sky.coflnet.com/api">https://sky.coflnet.com/api</a></p>
```
Comment (Ekwav) [inline src/pages/About/About.tsx:45]:
```suggestion
                    <p><a href="https://sky.coflnet.com/api">https://sky.coflnet.com/api</a></p>
<p>If you use it please link to <a href="https://sky.coflnet.com/data">https://sky.coflnet.com/data</a></p>
```

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

File: src/pages/About/About.tsx [hunk 5]
```diff
@@ -40,15 +40,43 @@ function ItemDetails() {
...
+<p>If you use it please link to <a href="https://sky.coflnet.com/data">https://sky.coflnet.com/data</a></p>
+                    <hr />
+<h2>Credits</h2>
+                    <h3>Player Heads</h3>
+                    <p>Are provided by<a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
```
Comment (Ekwav) [inline src/pages/About/About.tsx:50]:
```suggestion
                    <p>Are provided by <a href="https://mc-heads.net/">mc-heads</a> and <a href="https://craftatar.com">craftatar</a></p>
```

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-377.flou21.de

**Pull request merged**